packages/core/src/utils-hoist/instrument/fetch.ts

```diff
@@ -107,6 +107,25 @@ function instrumentFetch(onFetchResolved?: (response: Response) => void, skipNat
             addNonEnumerableProperty(error, 'framesToPop', 1);
           }
 
+          // We enhance the not-so-helpful "Failed to fetch" error messages with the host
+          // Possible messages we handle here:
+          // * "Failed to fetch" (chromium)
+          // * "Load failed" (webkit)
+          // * "NetworkError when attempting to fetch resource." (firefox)
+          if (
+            error instanceof TypeError &&
+            (error.message === 'Failed to fetch' ||
+              error.message === 'Load failed' ||
+              error.message === 'NetworkError when attempting to fetch resource.')
+          ) {
+            try {
+              const url = new URL(handlerData.fetchData.url);
+              error.message = `${error.message} (${url.host})`;
```

Comment:
I wonder whether we should limit to the domain (eg `example.com`) instead of the entire host (`foo.bar.example.com`). Domain could get pretty noisy in multi-tenant situations.

Reply:
Ah I forgot. Message doesn't contribute to grouping if there is a stack trace. Then it's probably fine/better.

Reply:
Hmm, so basically `url.host.split('.).slice(-2).join('.')`, to just get the last part? 🤔

Reply:
I adjusted this and also added a test for this!